From d700470379ab69f13948efcfbddb6bf373073a48 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 26 May 2010 11:16:13 +0000
Subject: [PATCH] [BUILTIN] Use TMPDIR in mkbuiltins

while merging upstream dash in klibc,
noticed that klibc dash had grown that useful feature.

Signed-off-by: maximilian attems <max@stro.at>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit a650e91b6e0290a6e67cd0648d86ba9e1e4a67aa)

Conflicts:

	ChangeLog
---
 src/mkbuiltins |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/mkbuiltins b/src/mkbuiltins
index 064cb76..e38ce4c 100644
--- a/src/mkbuiltins
+++ b/src/mkbuiltins
@@ -37,7 +37,7 @@
 
 tempfile=tempfile
 if ! type tempfile > /dev/null 2>&1; then
-	tempfile='mktemp /tmp/mkbuiltins.XXXXXX'
+	tempfile="mktemp ${TMPDIR:-/tmp}/builtin.XXXXXX"
 fi
 
 trap 'rm -f $temp $temp2' EXIT
-- 
1.6.0.3

